blog/2023-07-31-new-pre-post-command-plugin-hooks.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: "New pre and post command plugin hooks"
+slug: "2023-07-05-latest-conda-release-includes-new-plugin-hooks"
+description: |
+  The latest release of conda introduces two new plugin hooks: pre and post command. These hooks allow plugin
+  authors to execute code before and after any conda command. Learn more about how and why you would use
+  these in this post.
+authors: [travishathaway]
+tags: [conda, plugins]
+image: img/blog/2023-07-31-new-pre-post-command-plugin-hooks/pre-post-command-image.jpg
+---
+
+[conda-issue]: https://github.com/conda/conda/issues/12138
+[conda-guard]: https://github.com/travishathaway/conda-guard
+[conda-plugin-docs]: https://docs.conda.io/projects/conda/en/stable/dev-guide/plugins/index.html
+[matrix]: https://matrix.to/#/#conda:matrix.org
+
+import Image from '@theme/IdealImage';
+import banner from '@site/static/img/blog/2023-07-31-new-pre-post-command-plugin-hooks/pre-post-command-image.jpg';
+
+<Image img={banner} alt="Banner image for blog post; New pre and post command plugin hooks - picture of a plugin outlet"/>
+
+<p>
+    <i>
+        <small>
+            Image credit: <a href="https://unsplash.com/photos/7d8pxcMVl7A">Kier in Sight Archives on Unsplash</a>
+        </small>
+    </i>
+</p>
+
+With the latest conda release (`23.7.0` at the time this post was written), the ability to define
+two new plugin hooks was introduced: "pre command" and "post command". These two new plugin hooks give
+plugin authors the ability execute code before and after conda commands are run. In this blog post,
+we provide more details on how and why you may use these to extend the default behavior of conda.
+
+To see a fully functioning example of how to use these new hooks, please check out the [conda-guard
+project][conda-guard].
+
+<!-- truncate -->
+
+To explain how these plugin hooks can be used, we walk through two examples.
```

Comment:
```suggestion
To explain how these plugin hooks can be used, let's walk through two examples.
```

Reply:
Take or leave this suggestion, it's mostly a "conversational style" change

Reply:
I think I'm going to change it to `... we will cover...`